### ISSUE TITLE
Query parameters don't get converted to appropriate GraphQL types

### ISSUE THREAD

**Issue by thecodeboss** [sisense/graphql2rest#41]
## Description

When passing arguments in a query string, there is no type conversion step. So strings don't get converted into integers, etc. For example, an HTTP request like this one:
```http
GET https://myapi.com/projects?limit=100
```
would attempt to pass the `limit` parameter as a string `"100"` instead of consulting the GraphQL schema and realizing it is supposed to take an `Int` type instead.

It's possible to define a middleware that converts these, however I'd like to suggest it as a feature instead. Most GraphQL built-in scalar types I think could be automatically converted, but obviously custom scalars or non-scalars would still require custom middleware to be built.

Another workaround is to use the request body instead of the query string, as JSON allows you to specify scalars without ambiguity.

I would be happy to take a stab at creating a pull request for this if there's no immediate objection to the idea.

## Steps to Reproduce

Set up a `graphql2rest` project with the following GraphQL Schema:
```graphql
type Project {
  id: ID!
}

type Query {
  projects(limit: Int): [Project!]!
}
```
this `manifest.json`:
```json
{
  "endpoints": {
    "/projects": {
      "get": {
        "operation": "projects"
      }
    }
  }
}
```
and then execute this HTTP request:
```http
GET https://myapi.com/projects?limit=100
```
You should get a GraphQL error like the following:
```
Variable "$limit" got invalid value "100"; Expected type Int. Int cannot represent non-integer value: "100"
```

**Comment by roy-mor** [comment]
Thanks for this issue @thecodeboss. 

Just to clarify - what you specified under "You should get a GraphQL error like the following" is the current behavior, yes? Which is a problem because the API user specified an integer value for the query parameter, but it got interpreted as a string, thereby violating the schema and so the input validation failed, resulting in an API request with valid input failing?

**Comment by MatthewHerbst** [comment]
> Just to clarify - what you specified under "You should get a GraphQL error like the following" is the current behavior, yes? Which is a problem because the API user specified an integer value for the query parameter, but it got interpreted as a string, thereby violating the schema and so the input validation failed, resulting in an API request with valid input failing to execute?

Yes, that's the current behavior. The steps above are listed under a `Steps to Reproduce` heading

**Comment by thecodeboss** [comment]
> Just to clarify - what you specified under "You should get a GraphQL error like the following" is the current behavior, yes?

Yes

**Comment by roy-mor** [comment]
@thecodeboss OK good. I wanted to make sure. Please go ahead and create a pull request for this. Please make sure to add appropriate tests for your change. Thank you!

**Comment by roy-mor** [comment]
@thecodeboss any progress on your suggested PR?

**Comment by shuffledex** [comment]
We need that PR ASAP!
